APE14.rst

```diff
@@ -406,6 +426,74 @@ dimension encoding time-of-observation.
     wcs.world_axis_object_components = [('spec', 0, 'value')]
     wcs.world_axis_object_classes  = {'spec':('astropy.units.pixel': {})}
 
+Pixel and world coordinate ordering
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The API above provides a way to distinguish between pixel coordinates defined
+using the standard Cartesian ordering (x, y) and array indices defined using the
+(row, column) ordering. For example, values returned from
```

Comment:
(minor): ``(row, column) ordering`` -> ``row-major ordering (i.e. row, column)``?

Reply:
Agree

Reply:
Fixed in 0b34b6ccb10618c2ebdb2c24e416d2df0bb5014f